lyaemu/flux_power.py

```diff
@@ -113,13 +75,14 @@ def __init__(self, numlos = 32000, max_z= 4.2, max_k = 5.):
         #and hence does not commute with mean flux rescaling.
         #I have checked that for k < 0.1 the corrected version
         #is identical to the unsmoothed version (without mean flux rescaling).
-        self.spec_res = 0.
+        self.spec_res = 127
```

Comment:
Please note the comment: "hence does not commute with mean flux rescaling": it doesn't make sense to correct for spectral resolution at the same time as doing mean flux rescaling, unfortunately, because they are both non-linear operations. So you have to do the spectral resolution correction on the observations but not the simulations.